lib/chewy/journal.rb

```diff
@@ -61,16 +61,20 @@ def apply_changes_from(time)
       end
 
       def entries_from(time)
+        # TODO: group data by index_name and type_name to group DB queries
         query = {
-          filter: {
-            range: {
-              created_at: {
-                gte: time.to_i
-              }
+          range: {
+            created_at: {
+              gte: time.to_i
             }
           }
         }
-        Chewy.client.search(index: index_name, type: type_name, body: query, sort: 'created_at')['hits']['hits'].map { |r| r['_source'] }
+        size = Chewy.client.count(index: index_name, type: type_name, body: {query: query})['count']
```

Comment:
because count doesn't accept `filter`: `QueryParsingException[[some_prefix_chewy_journal] request does not support [filter]]`